README.md

```diff
@@ -6,6 +6,8 @@
 
 Dynamoose is a modeling tool for Amazon's DynamoDB (inspired by [Mongoose](http://mongoosejs.com/))
 
+Documentation: https://dynamoosejs.com
```

Comment:
Currently the website is completely reflective of v1, and doesn't reflect v2 at all. You can find the documentation in the `/docs` folder of the project.